### PR TITLE
Autoformat all code with Black

### PR DESCRIPTION
**Pull request by Xyene**
`black . --line-length 120 --skip-string-normalization`

### CODE REVIEW

File: dmoj/checkers/unordered.py [hunk 1]
```diff
@@ -4,6 +4,7 @@
 
 
 def check(process_output: bytes, judge_output: bytes, **kwargs) -> bool:
-    warnings.warn('`unordered` checker is deprecated, use `sorted` with '
-                  '`split_on` parameter instead', DeprecationWarning)
+    warnings.warn(
+        '`unordered` checker is deprecated, use `sorted` with ' '`split_on` parameter instead', DeprecationWarning
```
Comment (kiritofeng) [inline dmoj/checkers/unordered.py]: Should be single string.

File: dmoj/config.py [hunk 2]
```diff
@@ -77,15 +77,17 @@ def iteritems(self):
         if not hasattr(self.raw_config, 'items'):
             raise InvalidInitException('config node is not a dict')
         for key, value in self.raw_config.items():
-            yield key, ConfigNode(value, self, dynamic=self.dynamic) \
-                if isinstance(value, list) or isinstance(value, dict) else value
+            yield key, ConfigNode(value, self, dynamic=self.dynamic) if isinstance(value, list) or isinstance(
+                value, dict
```
Comment (kiritofeng) [inline dmoj/config.py]: This feels really jank

File: dmoj/cptbox/syscalls/generate_linux.py [hunk 3]
```diff
@@ -24,8 +24,9 @@
             name = 'fstatat'
         print('%d\t%s' % (int(syscall[0]), name), file=x86)
 
-with open('linux-x64.tbl', 'w') as x64, open('linux-x32.tbl', 'w') as x32, \
-        utf8reader(urlopen(LINUX_SYSCALLS_64)) as data:
+with open('linux-x64.tbl', 'w') as x64, open('linux-x32.tbl', 'w') as x32, utf8reader(
+    urlopen(LINUX_SYSCALLS_64)
```
Comment (kiritofeng) [inline dmoj/cptbox/syscalls/generate_linux.py]: Again feels jank

File: dmoj/executors/PIKE.py [hunk 4]
```diff
@@ -6,8 +6,13 @@ class Executor(ScriptExecutor):
...
     command = 'pike'
-    syscalls = ['epoll_create', 'epoll_ctl', ('fstatfs', ACCESS_EACCES),  # Linux
-                'socketpair', ('fpathconf', ACCESS_EACCES)]               # FreeBSD
+    syscalls = [
+        'epoll_create',
+        'epoll_ctl',
+        ('fstatfs', ACCESS_EACCES),  # Linux
+        'socketpair',
+        ('fpathconf', ACCESS_EACCES),
+    ]  # FreeBSD
```
Comment (kiritofeng) [inline dmoj/executors/PIKE.py]: Comment in wrong place

File: dmoj/judgeenv.py [hunk 5]
```diff
@@ -49,10 +53,7 @@
 
 
 def load_env(cli=False, testsuite=False):  # pragma: no cover
-    global problem_dirs, only_executors, exclude_executors, log_file, server_host, \
-        server_port, no_ansi, no_ansi_emu, env, startup_warnings, no_watchdog, \
-        problem_regex, case_regex, api_listen, secure, no_cert_check, cert_store, \
-        problem_watches, cli_command
+    global problem_dirs, only_executors, exclude_executors, log_file, server_host, server_port, no_ansi, no_ansi_emu, env, startup_warnings, no_watchdog, problem_regex, case_regex, api_listen, secure, no_cert_check, cert_store, problem_watches, cli_command
```
Comment (kiritofeng) [inline dmoj/judgeenv.py]: Isn't this significantly longer than 120 char...

File: dmoj/judgeenv.py [hunk 6]
```diff
@@ -143,8 +147,9 @@ def load_env(cli=False, testsuite=False):  # pragma: no cover
         problem_dirs = env.problem_storage_root
         if problem_dirs is None:
             if not testsuite:
-                raise SystemExit('problem_storage_root not specified in "%s"; '
-                                 'no problems available to grade' % model_file)
+                raise SystemExit(
+                    'problem_storage_root not specified in "%s"; ' 'no problems available to grade' % model_file
```
Comment (kiritofeng) [inline dmoj/judgeenv.py]: Should be a single string

File: setup.py [hunk 7]
```diff
@@ -27,6 +27,7 @@
     from Cython.Build import cythonize
 except ImportError:
     if has_pyx:
+        traceback.print_exc()
```
Comment (kiritofeng) [inline setup.py]: This line is new.

File: dmoj/judgeenv.py (hunk 5 above)
Comment (Xyene) [reply dmoj/judgeenv.py]: Black makes an explicit exception for `global`

File: setup.py (hunk 7 above)
Comment (Xyene) [reply setup.py]: This line is my debugging stupid CI failures :(

Comment (Xyene) [reply setup.py:30]: The issue has been fixed, but I'm going to leave this in here since it's more explicit/helpful.

**Pull request merged**